hermes-management/src/main/java/pl/allegro/tech/hermes/management/config/storage/StorageConfiguration.java

```diff
@@ -172,12 +177,15 @@ OfflineRetransmissionRepository zookeeperOfflineRetransmissionRepository() {
 
     @PostConstruct
     public void init() {
+        logger.info("Before ensuring init path exists");
         ensureInitPathExists();
+        logger.info("After ensuring init path exists");
     }
 
     private void ensureInitPathExists() {
         ZookeeperClientManager clientManager = clientManager();
         for (ZookeeperClient client : clientManager.getClients()) {
+            logger.info("Ensuring that path exists for client: " + client);
```

Comment:
I see that you log here a `client` object, which results in log like this:
```
2022-10-18 10:02:10.462 INFO  p.a.t.h.m.c.s.StorageConfiguration - Ensuring that path exists for client: pl.allegro.tech.hermes.management.infrastructure.zookeeper.ZookeeperClient@4a5498f
```

Maybe instead of logging object reference (*ZookeeperClient@4a5498f*), you can log `client.getDatacenterName()`? Something like:

```java
logger.info("Ensuring that path exists for Zookeeper client: {}", client.getDatacenterName());
```